libs/blocks/modal/modal.js

```diff
@@ -43,6 +43,7 @@ function closeModal(modal) {
 
   document.querySelectorAll(`#${id}`).forEach((mod) => {
     if (mod.nextElementSibling?.classList.contains('modal-curtain')) {
+      document.querySelector('body').style.removeProperty('overflow');
```

Comment:
I would do it like this:
```
document.body.style.removeProperty('overflow');
// and 
document.body.style.overflow = 'hidden';
slightly faster in my observation.

Reply:
Yes, makes sense, resolved here: https://github.com/adobecom/milo/pull/1419/commits/07dd951b1c25735d6a519b3e013f699a854e3a3d
thanks @Blainegunn !